onnx/backend/base.py

```diff
@@ -57,5 +57,5 @@ def run_model(cls, model, inputs, device, **kwargs):
         cls.prepare(model, device, **kwargs).run(inputs)
 
     @classmethod
-    def run_node(cls, node, inputs):
+    def run_node(cls, node, inputs, device='CPU'):
```

Comment:
It's better to be consistent with `run_model`, so either make `device` a required args or make `run_model` also has the default device. I understand the former is a breaking change, but hey we have a lot of breaking changes already right :-)

Reply:
Let me actually make device an optional arg, it's a better choice imho